plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/SyntheticColumnHandleBuilder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import static com.google.common.base.Splitter.fixedLength;
+
+public class SyntheticColumnHandleBuilder
+{
+    public static final int DEFAULT_COLUMN_ALIAS_LENGTH = 30;
+
+    public JdbcColumnHandle get(JdbcColumnHandle column, int nextSyntheticColumnId)
+    {
+        if (nextSyntheticColumnId < 0) {
+            throw new IllegalArgumentException("nextSyntheticColumnId rolled over and is not monothonically increasing any more");
```

Comment:
```suggestion
            throw new IllegalArgumentException("nextSyntheticColumnId rolled over and is not monotonically increasing any more");
```

Reply:
oh, also note we have following tools for such checks which you'll find used all over the code.
In this specific case `Verify.verify` does exactly what you want. Feel free to apply or ignore this comment since it's a nitpick.

- `Preconditions.checkState`
- `Verify.verify`